.github/workflows/test-runner-on-perf.yml

```diff
@@ -70,61 +70,38 @@ jobs:
+          GOOGLE_SA_CERT: 'jade-dev-account.pem'
+        with:
+          actions_subcommand: 'gcp_whitelist'
+          role_id: ${{ secrets.ROLE_ID }}
+          secret_id: ${{ secrets.SECRET_ID }}
+          google_project: broad-jade-perf
       - name: 'Checkout datarepo-helm-definitions repo'
         uses: actions/checkout@v2
         with:
           repository: 'broadinstitute/datarepo-helm-definitions'
-          token: ${{ secrets.HELM_REPO_TOKEN }}
+          token: ${{ secrets.BROADBOT_TOKEN }}
           path: datarepo-helm-definitions
       - name: "Update perf image tag with semVer"
         if: ${{ steps.read_property.outputs.LATEST_VERSION != steps.read_property.outputs.CURRENT_SEMVER }}
         uses: docker://mikefarah/yq:3.3.4
         with:
           args: yq w -i datarepo-helm-definitions/perf/datarepo/datarepo-api.yaml image.tag ${{ steps.read_property.outputs.LATEST_VERSION }}
-      - name: "Create datarepo-helm-definition pull request with updated perf image tag"
-        if: ${{ steps.read_property.outputs.LATEST_VERSION != steps.read_property.outputs.CURRENT_SEMVER }}
-        uses: broadinstitute/create-pull-request@v3.5.0 # forked from peter-evans/create-pull-request
-        with:
-          token: ${{ secrets.HELM_REPO_TOKEN }}
-          path: datarepo-helm-definitions
-          commit-message: "Perf Datarepo version update: ${{ steps.read_property.outputs.LATEST_VERSION }}"
-          committer: datarepo-bot <noreply@github.com>
-          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
-          title: "Perf Datarepo version update: ${{ steps.read_property.outputs.LATEST_VERSION }}"
-          branch: "version-update-${{ steps.read_property.outputs.LATEST_VERSION }}"
-          body: |
-            Update versions in perf env to reflect image tag ${{ steps.read_property.outputs.LATEST_VERSION }}.
-            *Note: This PR was opened by the [test-runner-perf GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
-          labels: "datarepo,automerge,version-update"
+      - name: "[datarepo-helm-definitions] Merge version update"
+        uses: broadinstitute/datarepo-actions/actions/merger@0.39.0
+        env:
+          COMMIT_MESSAGE:  "Perf Datarepo version update: ${{ steps.read_property.outputs.LATEST_VERSION }}"
+          GITHUB_REPO: datarepo-helm-definitions
+          SWITCH_DIRECTORIES: "true"
+          MERGE_BRANCH: master
       - name: "Install Helmfile"
         uses: broadinstitute/setup-helmfile@v0.6.0 #Forked from mamezou-tech/setup-helmfile
       - name: "Use helmfile to delete and reapply helm for api pod"
         run: |
           helmfile --version
           cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-helm-definitions/perf
-          echo "Killing perf api pod to force db migration (can be removed after DR-1544 is complete)"
-          helm delete -n perf perf-jade-datarepo-api
-          sleep 15
```

Comment:
Good catch!

Reply:
Thank you for facilitating this! Now we also shouldn't see so many "incidents" in our jade-alerts channel since we'll no longer take perf down on every run.